great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py

```diff
@@ -33,9 +34,12 @@
     parse_row_condition_string_pandas_engine,
     substitute_none_for_missing,
 )
+from great_expectations.validator.exception_info import ExceptionInfo
 from great_expectations.validator.metric_configuration import MetricConfiguration
 from great_expectations.validator.validation_graph import ValidationGraph
-from great_expectations.validator.validator import Validator
+
+logger = logging.getLogger(__name__)
+logging.captureWarnings(True)
```

Comment:
@donaldheppner Because we are now using  `graph.resolve_validation_graph()` correctly and consistently, which is defined as is currently in this module -- and used with the logging.  This correct usage is in "great_expectations/validator/metrics_calculator.py" starting on line 178.  Thanks.